pyuvsim/uvsim.py

```diff
@@ -532,6 +572,15 @@ def run_uvdata_uvsim(input_uv, beam_list, beam_dict=None, catalog=None, quiet=Fa
     vis_data.Free()
 
     if rank == 0:
+        if input_order is not None:
+            if len(input_order) < 2:
+                input_order = (input_order[0], None)
+
+            uv_container.reorder_blts(
+                order=input_order[0],
+                minor_order=input_order[1],
+            )
+
```

Comment:
So, here you reorder back to the original order if it was set. I can see two potential problems (which may not actually be problems:

1. If `input_order` is None, you don't re-order back to original. So the output order will be different from input. Like, if user puts in something with ridiculous arbitrary order, you just go ahead and reorder it and give it back in (time, baseline). Not sure there's any way around this, but it should at least be well-documented (or warned).
2. You reorder even if the input order was already (time, baseline). This is a waste of time, unless the reorder_blts function itself short-circuits it.

Reply:
We could issue a warning if the input order is not known and tell the  output will be (time, baseline). Or just scream it in the docstrings, or both.

good catch on 2, it looks like it does not short-circuit so that could be a decent waste of time.